src/main/scala/com/ibm/couchdb/api/builders/GetManyDocumentsQueryBuilder.scala

```diff
@@ -25,92 +25,143 @@ import upickle.default.write
 import scala.reflect.ClassTag
 import scalaz.concurrent.Task
 
-case class GetManyDocumentsQueryBuilder(
+sealed trait DocsInResult
+abstract class IncludeDoc[D: R] extends DocsInResult
+abstract class ExcludeDocs extends DocsInResult
+
+sealed trait MissingIdsInQuery
+trait MissingAllowed extends MissingIdsInQuery
```

Comment:
those that require type params have to be abstract classes I guess since traits can't have type params
 `MissingAllowed` and `MissingNotAllowed` don't require type params. 
Were you thinking of changing this for consistency?
Admitedly `ExcludeDocs` and `AnyDocs` can also be a traits